pkgs/glibc/build.textproto

```diff
@@ -65,5 +65,6 @@ build_step: {
 
 build_step: {
   argv: "rm"
+  argv: "-f"
```

Comment:
For some reason, this file doesn't exist when doing crossbuilding, so this arg is here to allow it to just continue on if it doesn't exist